man/man1/p11kmip.1.in

```diff
@@ -403,10 +413,10 @@ When none are set, an error occurs.
 .
 .TP 8
 .BR \-\-kmip\-client\-key\~\fIKEY\-\PATH\fP
-A path to the TLS client private key to use for the KMIP connection.
+A path to the TLS client key to use for the KMIP connection.
```

Comment:
I think we should leave `private` here, since this really is the private key of the client, used for authenticating the client to the server.